strongbox_test.go

```diff
@@ -109,21 +111,113 @@ func TestMain(m *testing.M) {
 		fmt.Fprintf(os.Stderr, "%v", string(out))
 		os.Exit(1)
 	}
-	out, err = command("/", "mkdir", HOME+"/test-proj")
+	out, err = command("/", "mkdir", defaultRepoDir)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "%v", string(out))
+		os.Exit(1)
+	}
+	out, err = command(defaultRepoDir, "git", "config", "--global", "init.defaultBranch", defaultBranch)
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "%v", string(out))
 		os.Exit(1)
 	}
-	out, err = command(HOME+"/test-proj", "git", "init")
+	out, err = command(defaultRepoDir, "git", "init")
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "%v", string(out))
 		os.Exit(1)
 	}
 	os.Exit(m.Run())
 }
 
+func TestMergeDriverMerge(t *testing.T) {
+	repoDir := defaultRepoDir
+	secDir := "secrets/dir0/"
+	secFileName := "sec0"
+	secFilePath := secDir + secFileName
+	secVal := "secret123wallaby"
+	_, keyID := keysFromKR(t, "test00")
+	ga := "secrets/* filter=strongbox diff=strongbox merge=strongbox"
+
+	assertWriteFile(t, repoDir+"/.gitattributes", []byte(ga), 0644)
+	assertWriteFile(t, repoDir+"/.strongbox-keyid", []byte(keyID), 0644)
+	assertCommand(t, repoDir, "mkdir", "-p", secDir)
+	assertWriteFile(t, repoDir+secFilePath, []byte(secVal), 0644)
+	assertCommand(t, repoDir, "git", "add", ".")
+	assertCommand(t, repoDir, "git", "commit", "-m", "\"TestMergeDriverMerge\"")
```

Comment:
Could avoid some escaping

```suggestion
	assertCommand(t, repoDir, "git", "commit", "-m", `"TestMergeDriverMerge"`)
```

Reply:
Can't merge on web UI - will make the change, ty

Reply:
this msg/code is already removed